### PR TITLE
Create ruleset: pop2imap.com

### PR DESCRIPTION
**Pull request by katzrkool**

### CODE REVIEW

File: src/chrome/content/rules/Pop2imap.com.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Pop2imap.com">
+	<target host="pop2imap.com" />
+	<target host="www.pop2imap.com" />
+
+	<rule from="^http:" to="https:" />
+	
+	<test url="http://www.pop2imap.com/how.php/" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Pop2imap.com.xml]: The test URL is not necessary here. Can you remove it please?

File: src/chrome/content/rules/Pop2imap.com.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!--
```
Comment (Bisaloo) [inline src/chrome/content/rules/Pop2imap.com.xml]: Please remove this comment since all targets are working over HTTPS

**Comment by katzrkool** [comment]
Made the requested changes

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**